openrtx/src/core/graphics.c

```diff
@@ -77,12 +82,17 @@ static const GFXfont fonts[] = { TomThumb,            // 5pt
                                  UbuntuRegular12pt7b, // 12pt
                                  UbuntuRegular16pt7b, // 16pt
                                  UbuntuRegular18pt7b, // 16pt
-                                 UbuntuRegular24pt7b  // 24pt
+                                 UbuntuRegular24pt7b,  // 24pt
     #else
     #error Unsupported font family!
     #endif
+                                 Symbols5pt7b,      // 5pt
+                                 Symbols6pt7b,      // 6pt
+                                 Symbols8pt7b       // 8pt
                                };
 
+static const uint8_t symbol_offset = SYMBOLS_SIZE_6PT - FONT_SIZE_6PT;
```

Comment:
Instead of creating the `symbol_offset` constant, I'd create a dedicated enum for the symbols. `symbolSize_t` or something like that.